i18n.go

```diff
@@ -91,6 +91,14 @@ func loadMessages(path string) {
 	if error := filepath.Walk(path, loadMessageFile); error != nil && !os.IsNotExist(error) {
 		ERROR.Println("Error reading messages files:", error)
 	}
+
+	// Read in messages from the modules
+	for _, module := range Modules {
```

Comment:
Messages is a different beast to begin with. It is loaded through an `init` method and `OnAppStart`. Templates are loaded in server.go:run. To me neither seem totally correct, exposing TemplatePaths seems unnecessary (as does ConfPaths and CodePaths). My thoughts are that the revel:addModule method should have a callBack list of items to be notified when a module is added, (with the possibility of adding more callbacks in the config). Each item then would take care of initializing its own area. However I do not want to even look at this until after we have broken completed the breaking up of revel. 

For the intermediate time though I think this is a valuable PR even if it is not implemented in the same way as the templates.